### PR TITLE
Hide subscription

### PR DESCRIPTION
**Pull request by nickpell**
@calellowitz cc: @kaapstorm

### CODE REVIEW

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -985,6 +985,11 @@ def _process_upgrade(domain, upgraded_privileges, new_plan_version, web_user):
             raise SubscriptionChangeError("The upgrade was not successful.")
 
 
+class SubscriptionBaseManager(models.Manager):
+    def get_queryset(self):
+        return super(SubscriptionBaseManager, self).get_queryset().filter(is_hidden_to_ops=False)
```
Comment (calellowitz) [inline corehq/apps/accounting/models.py]: More of a question than a code review comment, but is there a reason to use filter vs exclude?

**Comment by calellowitz** [comment]
:+1: when tests pass, question is obviously not blocking

File: corehq/apps/accounting/models.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/accounting/models.py]: I don't think there's a reason from a correctness/performance perspective, just which is easier to read (which might depend on the context and also be pretty subjective)

**Comment by nickpell** [comment]
:white_check_mark:

**Comment by calellowitz** [comment]
how is this set? or does this need to be done explicitly in the database?

**Comment by nickpell** [comment]
I'm not introducing a UI for this at the moment, until a more compelling use case comes up.  Right now this behavior is only requested for a small number of subscriptions, so it might not become necessary to provide subscription suppressing capabilities to Ops.

**Pull request merged**